platform/darwin/src/MGLStyle.h

```diff
@@ -610,6 +610,18 @@ MGL_EXPORT
  */
 @property (nonatomic, strong) MGLLight *light;
 
+#pragma mark Managing Style's localization
+
+/**
+ Enable it to determine the localization according to the languages
+ supported by the Vector Style source <a
+ href="https://www.mapbox.com/vector-tiles/mapbox-streets-v7/#overview">
+ Mapbox Streets</a>, it will look for the Preferred Language Order setting in
+ the system's Language & Region section of Settings. If it doesn't find a
+ match it will fall back to english localization.
```

Comment:
This property will currently only take effect in/after `-mapView:didFinishLoadingStyle:` — if this remains a limitation, we need to make this clear in the docs.